fvm/evm/emulator/signer.go

```diff
@@ -0,0 +1,21 @@
+package emulator
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+var DefaultBlockNumberForEVMRules = big.NewInt(1) // anything bigger than 0
```

Comment:
no need to export it?

Reply:
updated at [c848b92](https://github.com/onflow/flow-go/pull/4927/commits/c848b9212ca242dd68b095c4a3cd63058af8c377)